### ISSUE TITLE
Image shortcode for posts

### ISSUE THREAD

**Issue by falldowngoboone** [falldowngoboone/falldowngoboone-com#50]
I would like to build a shortcode for images that uses an agreed-upon contract to generate multiple size URLs, srcset, etc. It should allow for optional lazy loading, alt text, width, height and classes. It should produce a figure and the figure should have an optional caption. The figure can have multiple images.

**Comment by falldowngoboone** [comment]
It might be more beneficial to look at [`eleventy-img`](https://www.11ty.dev/docs/plugins/image/).

**Comment by falldowngoboone** [comment]
I would like to explore better margin alignment with page content as well. 100% full width?

Also, click to view full size.
